### DISCUSSION TITLE
Question: What kind of information telemetry option share?

### DISCUSSION THREAD

**Discussion by shariq1379** [microsoft-search/pnp-modern-search#2398] [Q&A] [answered]
### Check existing issues before logging new ones

There is a option to turn on/off telemetry with PnP Search Web parts. What kind of information it share  if we turn it on.
Search keywords/Search results/site collection info/ etc... ??? 

 
**Version used**
Ex: 4.7.0

**Comment by wobba** [comment, marked as answer]
It's just usage telemetry to see how much the PnP reusable controls are used. So not related to these web parts directly, but related to the PnP reusable controls we use in the UX. See https://pnp.github.io/sp-dev-fx-controls-react/#telemetry

**Reply by FranckyC** [reply]
Actually more information here https://github.com/pnp/telemetry-proxy-node.
